frontend/src/components/question-creation/QuestionSidebarItem.tsx

```diff
@@ -48,6 +49,11 @@ const QuestionSidebarItem = ({
             break;
           case QuestionElementType.MULTIPLE_CHOICE:
             setShowAddMultipleChoiceModal(true);
+            setIsMultiSelect(false);
+            break;
+          case QuestionElementType.MULTI_SELECT:
+            setShowAddMultipleChoiceModal(true);
+            setIsMultiSelect(true);
```

Comment:
I think we can just set one to true and if either are true, we can show the modal! i think you'll have to update `AddMultipleChoiceModal` where that logic is defined 🤔 
```suggestion
            setShowAddMultiSelectModal(true);
```